src/ncbitaxon.py

```diff
@@ -24,24 +25,27 @@
 related_synonym = "oboInOwl:hasRelatedSynonym"
 broad_synonym = "oboInOwl:hasBroadSynonym"
 
+# See OMO properties at
+# https://github.com/information-artifact-ontology/ontology-metadata/blob/master/src/templates/annotation_properties.tsv
 predicates = {
-    "acronym": broad_synonym,
-    "anamorph": related_synonym,
-    "blast name": related_synonym,
-    "common name": exact_synonym,
-    "equivalent name": exact_synonym,
-    "genbank acronym": broad_synonym,
-    "genbank anamorph": related_synonym,
-    "genbank common name": exact_synonym,
-    "genbank synonym": related_synonym,
-    "in-part": related_synonym,
-    "misnomer": related_synonym,
-    "misspelling": related_synonym,
-    "synonym": related_synonym,
-    "scientific name": exact_synonym,
-    "teleomorph": related_synonym,
+    "acronym": (broad_synonym, "OMO:0003000", "abbreviation"),
+    "anamorph": (related_synonym, None, None),
+    "blast name": (related_synonym, None, None),
+    "common name": (exact_synonym, "OMO:0003003", "layperson synonym"),
```

Comment:
Are you sure that "common name" and "layperson synonym" is the same?

Reply:
Yes, 99%. From reading examples, I think the intent was the same. Some examples:

| CURIE | Label | Common Name |
|------|------|------|
| [NCBITaxon:1046](https://ontobee.org/ontology/NCBITaxon?iri=http://purl.obolibrary.org/obo/NCBITaxon_1046) | Chromatiaceae | purple sulfur bacteria |
| [NCBITaxon:562](https://ontobee.org/ontology/NCBITaxon?iri=http://purl.obolibrary.org/obo/NCBITaxon_562) | Escherichia coli | E. coli |
| [NCBITaxon:9615](https://ontobee.org/ontology/NCBITaxon?iri=http://purl.obolibrary.org/obo/NCBITaxon_9615) | Canis lupus familiaris | dogs |

However, in a small number of instances, NCBI Taxonomy Database sometimes adds a very technical sounding synonym using this field like "agent of human granulocytic ehrlichiosis" for [NCBITaxon:948](https://ontobee.org/ontology/NCBITaxon?iri=http://purl.obolibrary.org/obo/NCBITaxon_948), but that's up to interpretation.

Reply:
Alright!